src/utils/components/NetworkInterfaceModal/utils/helpers.ts

```diff
@@ -146,7 +157,7 @@ export const getNadType = (nad: NetworkAttachmentDefinition): string => {
   try {
     const config = JSON.parse(nad?.spec?.config);
     //can be config.type or config.plugin first element only!'
-    return interfacesTypes?.[config?.type] || interfacesTypes?.[config?.plugins?.[0]?.type];
+    return interfaceTypes?.[config?.type] || interfaceTypes?.[config?.plugins?.[0]?.type];
```

Comment:
the fallback won't work because we have a proxy there

Reply:
It looks like this code existed before the proxy was introduced. I just changed the variable name of `interfacesTypes`. It's fortunate that you caught this, though. Would you mind checking through your PR that introduced the proxy change to make sure that it didn't compromise any other areas like this?

Reply:
The fame/blame goes to https://github.com/kubevirt-ui/kubevirt-plugin/pull/1424
I went through the code once again and it seems we are OK. However I would consider renaming that object to sth like `interfaceTypesProxy`  to make it more obvious that it's  not a plain object. The second proxy  `interfaceLabels` (that I've added following `interfacesTypes`) should probably also be renamed to keep both consistent.  Alternatively  we could replace both proxies with a regular method calls (in fact it's a hidden getter method call anyway).

Reply:
I'll leave that decision up to you since you added it.